package.json

```diff
@@ -10,7 +10,10 @@
   "scripts": {
     "story:dev": "pnpm --filter histoire story:dev",
     "build": "pnpm run -r --filter=!docs build",
-    "prepare": "pnpm simple-git-hooks"
+    "start": "cd packages/radix-vue && pnpm i && pnpm run build && cd ../../playground/vue3 && pnpm i && pnpm i ../../packages/radix-vue && pnpm run dev",
+    "prepare": "pnpm simple-git-hooks",
+    "lint": "eslint . --ignore-path .gitignore",
+    "lint:fix": "eslint . --fix --ignore-path .gitignore"
```

Comment:
I had this script earlier, but I wonder do we need to have this script since we have precommit hook already